x-pack/plugins/license_management/__jest__/__snapshots__/telemetry_opt_in.test.js.snap

```diff
@@ -40,7 +40,7 @@ exports[`TelemetryOptIn should display when telemetry not opted in 1`] = `
               Read more
             </EuiLink>
           }
-          className="licenseManagement__baseline"
+          className="eui-AlignBaseline"
```

Comment:
There's probably good reason for this that I'm not picking up as there are no styles attached to this class, but just noting that this naming format (prefix of eui-) is not one that I've seen elsewhere.